File: app/src/main/java/com/admin/theapp/logger/AppLogger.java
package com.admin.theapp.logger;

import android.support.annotation.NonNull;
import android.util.Log;

import com.theapp.tools.Logger;

public class AppLogger implements Logger {

    @Override
    public void d(@NonNull String tag, @NonNull String msg) {
        Log.d(tag, msg);
    }

    @Override
    public void e(@NonNull String tag, @NonNull String msg) {
        Log.e(tag, msg);
    }

    @Override
    public void e(@NonNull String tag, @NonNull String msg, @NonNull Throwable tr) {
        Log.e(tag, msg, tr);
    }

    @Override
    public void i(@NonNull String tag, @NonNull String msg) {
        Log.i(tag, msg);
    }

    @Override
    public void v(@NonNull String tag, @NonNull String msg) {
        Log.v(tag, msg);
    }

    @Override
    public void w(@NonNull String tag, @NonNull String msg) {
        Log.w(tag, msg);
    }
}

File: app/src/main/java/com/admin/theapp/logger/AppLogger.kt
package com.admin.theapp.logger

import android.util.Log

import com.theapp.tools.Logger

class AppLogger : Logger {

    override fun d(tag: String, msg: String) {
        Log.d(tag, msg)
    }

    override fun e(tag: String, msg: String) {
        Log.e(tag, msg)
    }

    override fun e(tag: String, msg: String, tr: Throwable) {
        Log.e(tag, msg, tr)
    }

    override fun i(tag: String, msg: String) {
        Log.i(tag, msg)
    }

    override fun v(tag: String, msg: String) {
        Log.v(tag, msg)
    }

    override fun w(tag: String, msg: String) {
        Log.w(tag, msg)
    }
}
